dashboard/DashboardDB.py

```diff
@@ -88,11 +102,9 @@ def update_daily_chours_usage(
             return
 
         table = self.get_daily_t_name(hpc)
-        day = (
-            day.strftime(self.date_format)
-            if day is not None
-            else date.today().strftime(self.date_format)
-        )
+
```

Comment:
No blank lines, these two assignments (table & day) belong to the with statement